custom_components/sleep_as_android/const.py

```diff
@@ -38,4 +38,5 @@
     "sound_event_baby",
     "sound_event_laugh",
     "before_alarm",
+    "alarm_rescheduled"
```

Comment:
Please add coma at the end of line.
Just to let future changes looks like one line changes: python is allowing this _(but JSON is not, so `en.json` will always have two lines change while adding just one)_.

Reply:
Updated.

Yeah, technically a trailing comma isn't spec'ful under JSON. 😸

Reply:
Oops, misread that comment - fixed and rebased.